assets/js/atomic/blocks/product-elements/rating/support.js

```diff
@@ -9,6 +9,16 @@ export const supports = {
 			text: true,
 			background: false,
 			link: false,
+			__experimentalSkipSerialization: true,
 		},
+		typography: {
+			fontSize: true,
+			__experimentalSkipSerialization: true,
+		},
+		spacing: {
+			margin: true,
+			__experimentalSkipSerialization: true,
+		},
```

Comment:
we need to provide additional guard for spacing feature with `hasSpacingStyleSupport`
```suggestion
		...( hasSpacingStyleSupport() && {
			spacing: {
				margin: true,
				__experimentalSkipSerialization: true,
			},
		} ),
```

Reply:
Thanks @tomasztunik, added!